docs/install-crossplane.md

```diff
@@ -160,6 +185,12 @@ kubectl delete -f stack-aws.yaml
 kubectl delete -f stack-azure.yaml
 ```
 
+#### Uninstalling Rook
```

Comment:
This is rendering like the `Uninstalling x` steps above were all changed if you look at rich diff, but I think git is correctly only viewing it as addition.